packages/c/convertutf/xmake.lua

```diff
@@ -0,0 +1,22 @@
+package("convertutf")
+    set_homepage("https://home.unicode.org/")
+    set_description("Conversions between UTF-32, UTF-16, and UTF-8. Circa 2004.")
+
+    set_urls("https://github.com/xmake-mirror/convertutf/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/xmake-mirror/convertutf.git")
+    add_versions("v1.0.1", "ecd79378cc9f24034a2edebf5f198e84c9eb8046261f8208908d2b29d4fa4d99")
+
+    on_install(function (package)
+        io.writefile("xmake.lua", [[
+            target("convertutf")
+                set_kind("$(kind)")
+                set_languages("c99")
+                add_files("ConvertUTF.c")
+                add_headerfiles("ConvertUTF.h")
+        ]])
```

Comment:
please add shared/static

https://github.com/xmake-io/xmake-repo/blob/47a1bd8a326ba23202adfb78797f2d3b115665f6/packages/z/zlib/xmake.lua#L66

Reply:
I don't believe it exports anything so it may be best to make it static only

Reply:
We can force to export it. see 

https://github.com/xmake-io/xmake-repo/blob/47a1bd8a326ba23202adfb78797f2d3b115665f6/packages/l/libimagequant/xmake.lua#L21-L23

Reply:
Oh, interesting. I'll give that a shot.